tutorials/pretraining-Vietnamese-data-curation/pretraining-Vietnamese-data-curation.ipynb

```diff
@@ -0,0 +1,1274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Processing High-Quality Vietnamese Data: Viettel’s Success with NVIDIA NeMo Curator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Open-source [large language models (LLMs)](https://www.nvidia.com/en-us/glossary/large-language-models/) excel in English but struggle with other languages, especially in Southeast Asia. This is primarily due to a lack of training data in these languages, limited understanding of local cultures, and insufficient tokens to capture unique linguistic structures and expressions. To fully meet customer needs, enterprises in non-English-speaking countries must go beyond generic models and customize them to capture the nuances of their local languages, ensuring a seamless and impactful customer experience.\n",
+    "\n",
+    "In this tutorial, we will use NeMo Curator to process high-quality [Vietnamese data](https://huggingface.co/datasets/VTSNLP/vietnamese_curated_dataset). We will guide you through the data curation pipeline used and share sample code for each stage."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites and Environment setups"
```

Comment:
Can you add a table of contents section to the tutorial? Similar to the one in the [image curation tutorial](https://github.com/hoangphu7122002/NeMo-Curator/blob/tutorials/pretraining-Vietnamese-data-curation/tutorials/image-curation/image-curation.ipynb)